src/cmdc_tools/datasets/official/MI/data.py

```diff
@@ -0,0 +1,50 @@
+import asyncio
+
+import pandas as pd
+import pyppeteer
+
+from ...base import DatasetBaseNoDate
+from ..base import CountyData
+
+
+class Michigan(DatasetBaseNoDate, CountyData):
+    def get(self):
+        urls = self._get_urls()
+        cases = self._get_cases(urls[0])
+
+        return cases
+
+    def _get_urls(self):
+        urls = asyncio.run(self._get_urls_async())
+        return urls
+
+    async def _get_urls_async(self):
+        browser = await pyppeteer.launch()
+        page = await browser.newPage()
+        await page.goto(
+            "https://www.michigan.gov/coronavirus/0,9753,7-406-98163_98173---,00.html"
```

Comment:
It looks like we can get more than just cases and deaths from this page. They also have a tab for testing which has positive/negative tests by date and county.

Reply:
Getting it now